FordLabs/retroquest
# Added intial support for a PWA (Progressive Web App).

## Overview
Adds the ability to use retroquest as a progressive web app.

## Testing Instructions
1. Go to the website and add it to your homescreen. When you load this homescreen app you should see retroquest being served without back / forward buttons or an address bar.
